### PR TITLE
Navigate to target page after login

### PR DESCRIPTION
**Pull request by thaigillespie**
Implement functionality allowing to navigate to a page after logging in from trying to access that page

Flow:

Step 1 (tries to navigate to officer/inductee/member-only page):
searchBarLocalhostEvent2

Step 2 (gets redirected to login page that will go to the officer/inductee/member-only page after logging in):
portal1

Step 3 (navigates to originally requested page after logging in with valid credentials):
portal3

### CODE REVIEW

**Comment by thaigillespie** [comment]
Ignore the weird formatting, my vscode changed the curly braces to be like that, and I'm currently trying to fix the issue where git commit hooks are not running when I commit my code : (

**Review by yuw006 — APPROVED** [review]
everything looks good, although can you add a couple notes for where the actual changes are made?

**Comment by thaigillespie** [comment]
> 
> 
> everything looks good, although can you add a couple notes for where the actual changes are made?

I don't think I understand what you mean so if you can clarify a bit more that would be great. What do you mean by "notes" here? And "actual changes"? The actual changes are in the files that I changed, no?

**Comment by thaigillespie** [comment]
@yuw006 @kyle1373 the styling issue is now resolved

File: frontend/src/pages/EventSignInPage/index.tsx
Comment (kyle1373) [inline frontend/src/pages/EventSignInPage/index.tsx:71]: Just curious, but I remember you telling me in my pull request that...

`Hosts:{' '}
{event.hosts
    .map(host => `${host.firstName} ${host.lastName}`)
    .join(', ')}
</Typography>`

should be changed to...

`Hosts: {event.hosts
    .map(host => `${host.firstName} ${host.lastName}`)
    .join(', ')}`

However, it looks like you have reverted that change. Did you change your mind about the formatting?

**Review by kyle1373 — APPROVED** [review]
LGTM! Just a quick question I had with EventSignInPage is all.

**Comment by yuw006** [comment]
> > everything looks good, although can you add a couple notes for where the actual changes are made?
> 
> I don't think I understand what you mean so if you can clarify a bit more that would be great. What do you mean by "notes" here? And "actual changes"? The actual changes are in the files that I changed, no?

oh i just meant to differentiate the code changes from the style ones, but since that's fixed it's all good.

File: frontend/src/pages/EventSignInPage/index.tsx
Comment (thaigillespie) [reply frontend/src/pages/EventSignInPage/index.tsx:71]: This is only because I made this branch before your changes were merged into master rercently, so I don't have the updates from master yet. You'll see when you scroll down to the end of this page that this branch is outdated. I was going to update it with changes from master once I get the necessary approvals. So I didn't change my mind about the formatting

Comment (kyle1373) [reply frontend/src/pages/EventSignInPage/index.tsx:71]: I see, thanks for the clarification!

**Pull request merged**
